### PR TITLE
Plugs the timeline/history component to the firebase database

### PR DESCRIPTION
**Pull request by lfaucon**

### CODE REVIEW

File: src/Activity/ChooseActivity/SessionTimeline.js [hunk 1]
```diff
@@ -25,42 +25,74 @@ function ThreeDotsIcon(props) {
...
+  { sessionRef: any },
+  { history: Object[] }
+> {
+  state = { history: {} };
+
+  componentWillMount() {
+    this.props.sessionRef.on("value", session => {
+      this.setState({ history: session.val() });
+    });
+  }
```
Comment (vood00child) [inline src/Activity/ChooseActivity/SessionTimeline.js]: This is considered legacy according to https://reactjs.org/docs/react-component.html. Moreover it clears the history the second time we look at it...

File: src/SessionHistory/index.js [hunk 2]
```diff
@@ -65,58 +64,84 @@ const icons = {
...
+const StyledSessionHistory = withStyles(styles)(SessionHistory)
+
+class FirebaseWrapper extends React.Component<{ sessionRef: any }, { history: Object[] }> {
+  state = { history: {} }
+
+  componentWillMount() {
+    this.props.sessionRef.on('value', session => {
+      this.setState({ history: session.val() })
+    })
+  }
```
Comment (vood00child) [inline src/SessionHistory/index.js]: same thing

**Comment by vood00child** [comment]
The session timeline and the history take time to load which is annoying for the user. Maybe we could add a callback that is triggered once the history is fetched from the db and only then we print the new view?

**Comment by lfaucon** [comment]
Thanks for catching that bug. If you think it's ok now we can merge it.

**Pull request merged**